core-bundle/contao/widgets/FileTree.php

```diff
@@ -381,11 +381,11 @@ protected function getPickerUrlExtras($values = array())
 	 *
 	 * @param File   $objFile
 	 * @param string $strInfo
-	 * @param string $blnRemovable
+	 * @param string $strClass
 	 *
 	 * @return string
 	 */
-	protected function getPreviewImage(File $objFile, $strInfo, $blnRemovable = false)
+	protected function getPreviewImage(File $objFile, $strInfo, $strClass='gimage')
```

Comment:
I decided to re-add the `gimage` class as there have been some encounters of the usage in e. g. the `tooltips-controller`.